components/nexus-blobstore-file/src/main/java/org/sonatype/nexus/blobstore/file/FileBlobStore.java

```diff
@@ -282,6 +310,18 @@ private void checkExists(final Path path, final BlobId blobId) throws IOExceptio
     }
   }
 
+  private String getPath(final Map<String, Map<String, Object>> attributes) {
+    return (String) attributes.get("file").get("path");
+  }
```

Comment:
This reminds me we should extract *AttributesMap to common so we can re-use that.